virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -36,8 +36,7 @@ include::modules/virt-creating-vm-custom-template.adoc[leveloffset=+1]
 
 Refer to the virtual machine fields section when creating a VM from the web console.
 
-// include::modules/virt-creating-vm-instancetype.adoc[leveloffset=+1]
-// uncomment this line when InstanceType becomes TP or GA
+include::modules/virt-creating-vm-instancetype.adoc[leveloffset=+1]
```

Comment:
The new procedure should appear after the reference modules (all the tables) because those apply to the "Create VM by customizing template". Then the order will make more sense.